common/config/rush/command-line.json

```diff
@@ -42,7 +42,8 @@
       "enableParallelism": true,
       "ignoreMissingScript": false,
       "ignoreDependencyOrder": false,
-      "allowWarningsInSuccessfulBuild": true
+      "allowWarningsInSuccessfulBuild": true,
+      "incremental": true
```

Comment:
this in itself, might get us faster consecutive buid:ci runs locally :)

Reply:
Yes! Also, unfortunately, it seems if warnings occur during a project build, the cache will not be written. So in testing time was still around 2 mins (down from about 5-6 in ADO) due to a few projects having to always be rebuilt. If we clear those up, then the build time should be seconds.